### DISCUSSION TITLE
Special character not displaying as expected

### DISCUSSION THREAD

**Discussion by B3N64** [cc-tweaked/CC-Tweaked#1716] [Q&A] [answered]
The characters in the following image are not displayed as expected when this command is executed with the character pasted in to it: term.write(""). 
![Unbenannt1](https://github.com/cc-tweaked/CC-Tweaked/assets/91955311/45ce8030-dc27-4b57-bc6e-83b2c79c036b)

The character "" = "\131"
![Unbenannt2](https://github.com/cc-tweaked/CC-Tweaked/assets/91955311/279c0dce-b027-4eea-8434-28578cc157fa)

My code:
```lua
term.clear()
term.setCursorPos(1, 1)
term.write((string.byte(""))) -- prints 194 instead of 131
term.setCursorPos(1, 2)
term.write("||")                     -- prints "\194" and "\131" instead of just "\131"
term.setCursorPos(1, 3)
term.write("|\131|")
term.setCursorPos(1, 4)
```

Output:
![Unbenannt](https://github.com/cc-tweaked/CC-Tweaked/assets/91955311/f1b83893-8199-45b2-9bd3-bb93c149dbb9)

**Comment by SquidDev** [comment, marked as answer]
Ahh. I suspect what's going on here is that your file is saved as UTF-8, but CC/Lua only works on plain bytes. This means that the Unicode codepoint is not saved as a literal `\131`, but instead [encoded](https://en.wikipedia.org/wiki/UTF-8#Encoding) as `\194\131`.

You could possibly configure your editor to save as latin1, but it's probably just easier to use escape codes (so `\131`) rather than pasting in the characters.
